source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards.xcodeproj/project.pbxproj

```diff
@@ -389,6 +397,16 @@
 /* End PBXFrameworksBuildPhase section */
 
 /* Begin PBXGroup section */
+		F4071C751FCCBAEF00AF4FEA /* json */ = {
+			isa = PBXGroup;
+			children = (
+				F4071C761FCCBAEF00AF4FEA /* json-forwards.h */,
```

Comment:
Why is this needed now?

Reply:
it was always there; i just changed directory representation in xcode solution file

---
In reply to: [153350249](https://github.com/Microsoft/AdaptiveCards/pull/992#discussion_r153350249) [](ancestors = 153350249)

Reply:
Ahh ok then 😄